src/uv-compat.h

```diff
@@ -0,0 +1,29 @@
+#include <nan.h>
+
+#if NAUV_UVVERSION < 0x10000
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <pthread.h>
+#endif
+
+NAN_INLINE int uv_thread_equal(const uv_thread_t* t1, const uv_thread_t* t2) {
+#ifdef _WIN32
+    return *t1 == *t2;
+#else
+    return pthread_equal(*t1, *t2);
+#endif
+}
+
+#if !NODE_VERSION_AT_LEAST(0, 9, 2)
```

Comment:
See https://github.com/libuv/libuv/commit/6591d000d13714c9e362942fbeaeca2b29507a25 and https://github.com/libuv/libuv/commit/a31024865b3146597cecc3d7f5bdf251da8b1a9b